e2e/tests/negative/checksum_enabled_large_volume_with_multiple_rebuilding.robot

```diff
@@ -26,7 +26,7 @@ Verify Large Volume Data Integrity During Replica Rebuilding with Recurring Jobs
     ...                - Delete a replica and wait for the replica rebuilding.
     ...                - Check volume data is intact
     Given Setting snapshot-data-integrity is set to enabled
-    And Setting snapshot-data-integrity-immediate-check-after-snapshot-creation is set to true
+    And Setting snapshot-data-integrity-immediate-check-after-snapshot-creation is set to {"v1":"true","v2":"true"}
```

Comment:
@derekbit If `snapshot-data-integrity-immediate-check-after-snapshot-creation` supports both `v1` and `v2`, do you know why set it to a simple value `true` won't automatically expand to `{"v1":"true","v2":"true"}` just like https://github.com/longhorn/longhorn-tests/blob/7bebd6a8d45adab63af50a413180664c2a011172/e2e/tests/regression/test_settings.robot#L25 and https://github.com/longhorn/longhorn/issues/10926#issuecomment-3201279967?